packages/taquito/src/contract/interface.ts

```diff
@@ -175,4 +179,24 @@ export interface ContractProvider extends StorageProvider {
   registerGlobalConstant(
     params: RegisterGlobalConstantParams
   ): Promise<RegisterGlobalConstantOperation>;
+
+  /**
+   *
+   * @description Originate a new tx rollup. Will sign and inject an operation using the current context
+   *
+   * @returns An operation handle with the result from the rpc node
+   *
+   * @param TxRollupOriginateParams Originate rollup operation parameter
+   */
+  originateTxRollup(params?: TxRollupOriginateParams): Promise<TxRollupOriginationOperation>;
```

Comment:
this might be personal preference, but because the other function interface name is `txRollupSubmitBatch`, maybe we could rename this one to be `txRollupOriginate` to have a consistent naming pattern? What do you think?

Reply:
Yes I was thinking the same at some point. I will make the change, thanks